src/services/submissionService.ts

```diff
@@ -158,6 +183,43 @@ export const updateSubmission = async (id: string, updatedFields: Partial<dbSche
     }
 }
 
+const validateSubmissionInputFields = (input: gqlType.SubmissionInput): Result<dbSchema.Submission> => {
+    const validatedSubmissionResult: Result<dbSchema.Submission> = {
+        hasErrors: false,
+        errors: []
+    }
+    
+    if (input.googleMapsUrl === null || input.googleMapsUrl === undefined || !input.googleMapsUrl.trim()) {
+        validatedSubmissionResult.hasErrors = true
+        validatedSubmissionResult.errors?.push({
+            field: 'googleMapsUrl',
+            errorCode: ErrorCode.MISSING_INPUT,
+            httpStatus: 400
+        })    
+    }
+
+    if (input.healthcareProfessionalName === null || input.healthcareProfessionalName === undefined || 
+        !input.healthcareProfessionalName.trim()) {
```

Comment:
We should put a limit on all these string lengths. 256 should be more than enough

Reply:
N/m ha just saw the next one. 

We should probably limit the map url to 1028